circle.yml

```diff
@@ -61,6 +61,11 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
+    - pyenv local 3.4.3:
+        parallel: true
+    - pip install -r docs-requirements.txt:
+        parallel: true
+    - rm -rf docs/API && make -C docs clean:
+        parallel: true
     - python setup.py docs:
         parallel: true
```

Comment:
Another option is to simply not run this on the effected Python version.  The docs are being build by Travis on three versions, and AppVeyor on two versions.  Circle can do one build instead of two, and the world will not end ;-)